inductiva/tasks/methods.py

```diff
@@ -98,10 +101,9 @@ def _fetch_tasks_from_api(
             raise e
 
 
-def get(
-    last_n: int = 5,
-    status: Optional[Union[str, models.TaskStatusCode]] = None
-) -> List["inductiva.tasks.Task"]:
+def get(last_n: int = 5,
+        status: Optional[Union[str, models.TaskStatusCode]] = None,
+        project: str = None) -> List["inductiva.tasks.Task"]:
```

Comment:
The project argument could also be a `Project` instance:

```suggestion
        project: Unido[str, Project] = None) -> List["inductiva.tasks.Task"]:
```